src/main/webapp/js/components/navbar.jsx

```diff
@@ -73,20 +67,15 @@ class Navbar extends Component {
                     <RoutePrivate path="/callLogCommunityPerson" component={CallLogCommunityPerson} />
                     <RoutePrivate path="/smsLogVaccinees" component={SmsLogVaccinees} />
                     <RoutePrivate path="/smsLogCommunityPerson" component={SmsLogCommunityPerson} />
-                    <RoutePrivate path="/role" component={Roles} />
                     <RoutePrivate path="/users" component={Users} />
                     <RoutePrivate path="/enrollmentGroup" component={Enrollment} />
                     <RoutePrivate path="/messaging" component={Messaging} />
                     <RoutePrivate path="/languages" component={Languages} />
-                    <RoutePrivate path="/vaccineeFields" component={VaccineeFields} />
-                    <RoutePrivate path="/keyCommunityPersonFields" component={KeyCommunityPersonFields} />
-                    <RoutePrivate path="/sitesFields" component={SiteFields} />
-                    <RoutePrivate path="/visitScheduleFields" component={VisitFields} />
                     <RoutePrivate path="/messageCampaign" component={MessageCampaign} />
                     <RoutePrivate path="/viewEntity/:entityType" component={ViewEntity} />
-                    <RoutePrivate path="/entityEdit/:entityType" component={EditEntityPage} />
-                    <RoutePrivate path={`/entityEdit/${ROLE_ENTITY}/:id`} component={RoleEditPage} />
+                    <RoutePrivate exact path="/entityEdit/:entityType" component={EditEntityPage} />
```

Comment:
you will not need the exact if you move it after the "/entityEdit/:entityType/:id"